### PR TITLE
Remove excess code

### PR DESCRIPTION
**Pull request by xichengliudui**
Remove excess code

### CODE REVIEW

File: pkg/controller/sync/version/adapter.go [hunk 1]
```diff
@@ -45,7 +45,6 @@ type VersionAdapter interface {
 func NewVersionAdapter(client fedclientset.Interface, namespaced bool) VersionAdapter {
 	if namespaced {
 		return newNamespacedVersionAdapter(client)
-	} else {
-		return newClusterVersionAdapter(client)
 	}
+	return newNamespacedVersionAdapter(client)
```
Comment (poothia) [inline pkg/controller/sync/version/adapter.go]: i think it should be "return newClusterVersionAdapter(client)"

**Comment by xichengliudui** [comment]
@poothia  Thank you for reminding me. I see the wrong

**Pull request merged**